packages/docs/pages/plugin/mention/index.tsx

```diff
@@ -197,6 +204,14 @@ export default function Mention(): ReactElement {
             not support a multi-character mentionTrigger.
           </span>
         </div>
+        <div className={styles.param}>
```

Comment:
There are now 2 `mentionTrigger` paragraphs. I guess you can delete the old one. You also need to add the new parameter for the `defaultSuggestionsFilter` function.

Reply:
I removed old one.

Reply:
Theres also the paragraph `In addition to the plugin the module exports defaultSuggestionsFilter. As first argument it takes the search term as a String. The second argument is an array of mentions. The function returns the filter list based on substring matches.`. Would be nice if you could add the new `trigger` parameter.